recipe/meta.yaml

```diff
@@ -64,7 +64,7 @@ requirements:
     # This will ensure that ncl works regardles of the gsl downloaded
     # (with or without openblas) at build time.
     # We can remove this once we have a better solution.
-    - openblas
+    - openblas 0.2.*
```

Comment:
Humm. I missed that when greping the file. I probably had a typo. Thanks @isuruf!